actions/admin/settings/180.dkim.php

```diff
@@ -30,6 +30,20 @@
 					'save_method' => 'storeSettingFieldInsertBindTask',
 					'overview_option' => true
 				),
+				'dkim_service_type' => array(
+					'label' => $lng['dkim']['dkim_service_type']['title'],
+
+					'settinggroup' => 'dkim',
+					'varname' => 'dkim_service_type',
+					'type' => 'option',
+					'default' => 'dkim-filter',
+					'option_mode' => 'one',
+					'option_options' => array(
+						'dkim-filter' => 'dkim-filter',
```

Comment:
don't know but on the contrary, I think it's better having both with a switch in Settings to turn each On / Off.

Or maybe an automatic setting that disables One when the other is Active; say disable opendkim while Rspamd is active.

At times, One might not always the a Full Power Horse and having something active just to utilize less than 20% of it's features to me can be a waste of resources.

Just my thought anyway especially since I have installed Froxlor for some people with limited Server resources or those just hosting very very handful of sites.